torch/_tensor_docs.py

```diff
@@ -2645,11 +2645,26 @@ def callable(a, b) -> number
 
 add_docstr_all('numpy',
                r"""
-numpy() -> numpy.ndarray
+numpy(force=False) -> numpy.ndarray
```

Comment:
You want to specify here that this is kwarg only?

Reply:
probably yes, but not sure how. Like this?
`numpy(*, force=False) -> numpy.ndarray`

Reply:
Yes